.github/workflows/cd.yml

```diff
@@ -0,0 +1,29 @@
+name: vue-vocabulary-cd
+
+on:
+  release:
+    types:
+    - published
+
+jobs:
+  deploy:
+    name: Deploy
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setup Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: '12.16.1'
+
+    - name: Install dependencies
+      run: npm install --no-audit
+
+    - name: Run build
+      run: npm run build
+
+    - name: Publish package
+      run: npm publish --access public
```

Comment:
A similar deploy script exists in this repo as well. Please sync changes with your other PR.

Reply:
Yes, I will update it now as the other one has been approved. Thank You :)